uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_009
# Avinash

## What does this PR do?  
This PR enhances the layout and design of the `profiles/show.html.haml` page by:  
- Aligning the profile section to the left and the actions section to the right using a responsive flexbox layout.  
- Updating the SCSS styles for both sections to achieve a modern and visually appealing design inspired by Apple.  
- Adding responsive behavior for better usability on smaller screens.  

## Related Issues  
No specific issues referenced. This enhancement improves the user interface as part of ongoing development.  

## Testing  
To test this PR, follow these steps:  
1. Navigate to the profile page (`/profile`) after logging in.  
2. Verify that the profile details (Username, Email, Edit Profile, Logout) are aligned to the left of the screen.  
3. Confirm that the actions list (Create Server, Your Servers, Items, View Inventory, View Transactions) is aligned to the right.  
4. Resize the browser window to ensure the layout stacks vertically on smaller screens (e.g., mobile devices).  
5. Verify the hover effects and the visual hierarchy of the links/buttons.  

## Checklist  

## Extra Information  
- Screenshot of the updated design:  
  *(Attach the screenshot showing the updated layout with the left and right sections for reference)*  
<img width="1512" alt="Screenshot 2024-11-17 at 9 05 27 PM" src="https://github.com/user-attachments/assets/feb77b0d-c408-458e-886b-8d0d20845f2a">
